com.teamspeak.TeamSpeak3.yaml

```diff
@@ -49,12 +49,12 @@ modules:
 #          url: https://teamspeak.com/en/downloads/#ts3client
 #          version-pattern: https://files.teamspeak-services.com/releases/client/3\.\d+\.\d+/TeamSpeak3-Client-linux_amd64-3\.(\d+\.\d+).run
 #          url-template: https://files.teamspeak-services.com/releases/client/3.$version/TeamSpeak3-Client-linux_amd64-3.$version.run
-#        x-checker-data:
-#          type: html
-#          url: http://versions.teamspeak.com/ts3-client-2
-#          version-pattern: "\u0006stable\u0010.*3\\.(\\d+\\.\\d+)\u0012"
-#          encoding-error: ignore  # requires https://github.com/flathub-infra/flatpak-external-data-checker/pull/410
-#          url-template: https://files.teamspeak-services.com/releases/client/3.$version/TeamSpeak3-Client-linux_amd64-3.$version.run
+        x-checker-data:
+          type: html
```

Comment:
Use the `anitya` checker. The teamspeak client has ID `8714` https://release-monitoring.org/project/8714/

Reply:
Not sure what that project is, but the content is pretty much trash. All the version numbers pretty much don't exist and the latest release is 3.6.2 and not 3.12.1. Their update check is as broken as the original one from this flatpak repo.

Reply:
OK, scrape https://files.teamspeak-services.com/releases/client/index.html with the HTML checker.

Reply:
I already implemented one for their website! Thanks!